src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -138,11 +142,15 @@ def __init__(
         self._quantize_embeddings = quantize_embeddings
         self._reduce_range = reduce_range
         self._quantize_linear_activations = quantize_linear_activations
+        self._enable_int4_activations = enable_int4_activations
 
         self._modules_to_quantize = None
         self._qat_enabled = False
         self._quantization_observer_disabled = False
         self._bn_stats_frozen = False
+        self._quant_min = 0
+        self._quant_max = 255
+        self._dtype = torch.quint8
```

Comment:
these shouldn't be stored here. instead the activation_kwargs should be updated conditionally in `_enable_module_qconfig`

Reply:
Do you mean `_enable_module_qat`?